src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -87,7 +87,7 @@ struct sof;
 #define SRAM_ALIAS_MASK		0xFF000000
 #define SRAM_ALIAS_OFFSET	SRAM_UNCACHED_ALIAS
 
-#if !defined UNIT_TEST && (CAVS_VERSION <= CAVS_VERSION_1_8 || !defined __ZEPHYR__)
+#if !defined UNIT_TEST
```

Comment:
I think we now want this for all cAVS versions on Zephyr? In other words this patch isn't needed any more?

Reply:
Sorry, @lyakh not following you, can you provide more context ?

Reply:
@lgirdwood hm, that was a long time ago, not sure what I meant, maybe I've mixed up `+` and `-`? looks like we do need this PR, let me approve it